R/aging-data.R

```diff
@@ -9,35 +9,35 @@ AgingData <- R6::R6Class("AgingData",
   active = list(
     #' @field individualIds Array of individual ids (corresponding to the individualId column of the aging table)
     individualIds = function(value) {
-      private$wrapIntegerProperty("IndividualIds", value)
+      private$.wrapProperty("IndividualIds", value, asInteger = TRUE)
     },
     #' @field parameterPaths Array of parameter paths (corresponding to the ParameterPath column of the aging table)
     parameterPaths = function(value) {
-      private$wrapProperty("ParameterPaths", value)
+      private$.wrapProperty("ParameterPaths", value)
     },
     #' @field times Array of time values (corresponding to the Time column of the aging table)
     times = function(value) {
-      private$wrapProperty("Times", value)
+      private$.wrapProperty("Times", value)
     },
     #' @field values Array of parameter values (corresponding to the Value column of the aging table)
     values = function(value) {
-      private$wrapProperty("Values", value)
+      private$.wrapProperty("Values", value)
     }
   ),
   public = list(
     #' @description
     #' Initialize a new instance of the class
     #' @return A new `OSPSuite.R.Domain.AgingData` object.
     initialize = function() {
-      ref <- rClr::clrNew("OSPSuite.R.Domain.AgingData")
-      super$initialize(ref)
+      netObj <- rSharp::newObjectFromName("OSPSuite.R.Domain.AgingData")
+      super$initialize(netObj)
```

Comment:
The `NetObject` class from rSharp requires a pointer to .NET object in the constructor. `DotNetWrapper` extends it for accepting valid `NetObject` instances.